### PR TITLE
math-magicians-project

### PR DESCRIPTION
**Pull request by micheaol**
##In this pull request:

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Create a route for every page
- Make sure that navigation links for all 3 routes are displayed on each page.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
```
Comment (mikenath223) [inline package.json:13]: - [ ] Kindly deploy the project online and include the live deploy link in the readme.

File: src/components/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable max-len */
```
Comment (mikenath223) [inline src/components/Home.jsx]: - [ ] Kindly remove the eslint disable line and fix the issue in the code.

**Review by er-contreras — APPROVED** [review]
Hi @micheaol , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**